examples/basic_flow/Makefile

```
@@ -1,9 +1,10 @@
 AZBI := epiphanyplatform/azbi:0.0.1
 AZKS := epiphanyplatform/azks:0.0.1
 
+makefile_dir_path := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
```

This is the "optimized" way (uses built-in functions). :+1:
```suggestion
makefile_dir_path := $(patsubst %/,%,$(dir $(realpath $(firstword $(MAKEFILE_LIST)))))
```